src/audio/google_audio_post_processing_mock.c

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
```

Comment:
I'm pretty sure this is not the correct path to place the mock files. Maybe: ./test/ directory can be used for that?

Reply:
I disagree, that is for running tests, these are not tests but stand ins that can be used during development